README.md

```diff
@@ -19,16 +19,16 @@ Properties
 -   **chunk_size**: Number of signals to notify each *interval* period and *emit* command.
 -   **interval**: Period at which signals are notified from queues.
 -   **backup_interval**: Period at which queues are backed up to disk using persistance. Queues are also backed up on stop.
--   **reload** (default=False): If True, notified signals immediately get reloaded back onto the end of the queue they came off of.
--   **uniqueness**: Expression Property. If specified, each queue (i.e. *group_by*) will not allow multiple signals that evaluate to the same *uniqueness*. If a signal comes in that matches *group_by* and *uniqueness* with a signal already in the queue, then it the new signal dropped.
+-   **auto-reload** (default=False): If True, notified signals immediately get reloaded back onto the end of the queue they came off of.
+-   **uniqueness**: Expression Property. If specified, each queue group (i.e. *group_by*) will not allow multiple signals that evaluate to the same *uniqueness*. If an incoming signal matches *group_by* and *uniqueness* with a signal already in the queue, then the new signal is dropped.
 
 Dependencies
 ----------------
 None
 
 Commands
 ----------------
-
+Use the nio_command block to issue the following commands:
```

Comment:
Remove this line. That is one way to use the commands but it does not belong in the docs for each block. This kind of documentation would go well somewhere here: https://github.com/nioinnovation/docs